src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/IssueAttribute.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Infrastructure.Common
+{
+    // The [Issue] attribute can be applied to any test method that
+    // is marked with [WcfFact] or [WcfTheory].  It indicates the open
+    // issue number and optionally the platform(s) to which it applies.
+    // Examples: 
+    //  [Issue(999)]
+    //  [Issue(999, PlatformID.OSX)]
+
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    public sealed class IssueAttribute : WcfSkippableAttribute
+    {
+        public int Issue { get; private set; }
+
+        public PlatformID PlatformID { get; private set; }
+
+        public IssueAttribute(int issue, PlatformID platformID)
```

Comment:
Is there any benefit to having a repo name here? Or can we assume that we will always have a tracking bug in our repo so all Issue id's will be for dotnet/wcf?